.github/workflows/build.yml

```diff
@@ -17,6 +17,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
```

Comment:
Can we use the `dart-lang/setup-dart` instead of the flutter one to reduce the setup time of the flutter?

Reply:
That's what I started with, seems like it won't work with just setup-dart:

https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/actions/runs/6559770182/job/17816029593

<img width="1036" alt="Screenshot 2023-10-18 at 13 03 51" src="https://github.com/AgoraIO-Extensions/Agora-Flutter-SDK/assets/5754073/e38c29f4-430b-492d-a77d-cb21ec6ac25d">